cluster/gke/config-test.sh

```diff
@@ -20,6 +20,7 @@ NETWORK=${KUBE_GKE_NETWORK:-e2e}
 NODE_TAG="k8s-${CLUSTER_NAME}-node"
 IMAGE_TYPE="${KUBE_GKE_IMAGE_TYPE:-container_vm}"
 ENABLE_KUBERNETES_ALPHA="${KUBE_GKE_ENABLE_KUBERNETES_ALPHA:-}"
+NODE_OS_DISTRIBUTION=gci
```

Comment:
Agree that it needs some logic to handle different images.

I think there may be an inconsistency between how IMAGE_TYPE is handled between cluster/gke/util.sh and cluster/gke/config-test.sh.

If IMAGE_TYPE is unset when cluster/gke/util.sh::kube-up() is called, there is no default value.  However, the cluster is provisioned with gci images.
If IMAGE_TYPE is unset when cluster/test-e2e.sh is called, the default value is "container-vm".  IMAGE_TYPE is ignored for testing anyway, so it doesn't seem to affect anything.

Changing IMAGE_TYPE default value to gci seems to be a good idea.  If unset, it will match the cluster.  If set, it will also match the cluster.